packages/caravan-psbt/src/psbtv2/psbtv2.ts

```diff
@@ -1326,6 +1333,8 @@ export class PsbtV2 extends PsbtV2Maps {
         .readInt32LE(0);
     }
 
+    psbtv2.PSBT_GLOBAL_FALLBACK_LOCKTIME = psbtv0.locktime;
```

Comment:
This was just missing, but resulting unsigned raw transactions constructed by `toV0` were not matching the text vector unsigned raw txn.